lib/rails/diff.rb

```diff
@@ -61,6 +61,16 @@ def rails_path
         end
       end
 
+      def railsrc_path
+        "#{ENV["HOME"]}/.railsrc"
+      end
+
+      def railsrc_options
+        return [] unless File.exist?(railsrc_path)
+
+        File.read(railsrc_path).split("\n")
```

Comment:
Would this be equivalent?
```suggestion
        File.read(railsrc_path).lines
```

Reply:
We're splitting then joining by spaces later. Perhaps we could return
```suggestion
        File.read(railsrc_path).tr("\n", " ")
```

Reply:
Great, suggestion thanks! Updated 👍🏼